examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,231 @@ Documentation for other releases can be found at
-
-```sh
-$ kubectl create -f examples/mysql-wordpress-pd/mysql.yaml
-```
+## Create the MySQL Password Secret
 
-It may take a short period before the new pod reaches the `Running` state.
-List all pods to see the status of this new pod and the cluster node that it is running on:
+Use a [Secret](../../docs/user-guide/secrets.md) object to store the
+MySQL password. This is referenced by the MySQL and WordPress pods when
+they are created. First create a temporary file called `password.txt`
+and save your password in it. Make sure to not have a trailing newline
+at the end of the password. Then, create the Secret object.
 
-```sh
-$ kubectl get pods
+```shell
+kubectl create secret generic mysql-pass --from-file=password.txt
 ```
 
+## Deploy MySQL
 
-#### Check the running pod on the Compute instance
+Now that the persistent disks and secrets are defined, the Kubernetes
+pods can be launched. We'll start with MySQL using
+[mysql-deployment.yaml](mysql-deployment.yaml).
 
-You can take a look at the logs for a pod by using `kubectl.sh log`.  For example:
-
-```sh
-$ kubectl logs mysql
+```shell
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/mysql-deployment.yaml
 ```
 
-If you want to do deeper troubleshooting, e.g. if it seems a container is not staying up, you can also ssh in to the node that a pod is running on.  There, you can run `sudo -s`, then `docker ps -a` to see all the containers.  You can then inspect the logs of containers that have exited, via `docker logs <container_id>`.  (You can also find some relevant logs under `/var/log`, e.g. `docker.log` and `kubelet.log`).
-
-### Start the Mysql service
-
-We'll define and start a [service](../../docs/user-guide/services.md) that lets other pods access the mysql database on a known port and host.
-We will specifically name the service `mysql`.  This will let us leverage the support for [Docker-links-compatible](../../docs/user-guide/services.md#how-do-they-work) service environment variables when we set up the wordpress pod. The wordpress Docker image expects to be linked to a mysql container named `mysql`, as you can see in the "How to use this image" section on the wordpress docker hub [page](https://registry.hub.docker.com/_/wordpress/).
+Note that we've defined a volume mount for `/var/lib/mysql`, and then
+created a Persistent Volume Claim that looks for a 20G volume. This
+claim is satisfied by any volume that meets the requirements, in our
+case one of the volumes we created above.
 
-So if we label our Kubernetes mysql service `mysql`, the wordpress pod will be able to use the Docker-links-compatible environment variables, defined by Kubernetes, to connect to the database.
+Also note that we specified the password by referencing the secret
+`mysql-pass` that we created above. Secrets can have multiple
```

Comment:
Would be helpful to explicitly state where / how it was referenced - in the mysql-deployment.yaml env section.

May something like..."The password is provided to the mysql container as an environment variable called `MYSQL_ROOT_PASSWORD`.  This is defined in the `env` section in mysql-deployment.yaml"